auth/user-differentiation.md

```diff
@@ -285,6 +296,8 @@ final class AuthorizationListener
     public function __invoke(MvcAuthEvent $mvcAuthEvent)
     {
         $authorization = $mvcAuthEvent->getAuthorizationService();
+        
+        $authorization->addRole('user');
```

Comment:
Can you add a comment on what this role may mean? (magic constants are problematic, even in examples)

Reply:
The example code fails with a fatal error about missing user role "user" when executing it:

[https://discourse.laminas.dev/t/role-user-not-found-following-authorizations-docs/1397](https://discourse.laminas.dev/t/role-user-not-found-following-authorizations-docs/1397)